server/services/team/service.ts

```diff
@@ -475,7 +475,7 @@ export class TeamService extends ServiceClass implements ITeamService {
 		return this.TeamMembersModel.findByTeamIds(teamIds, options).toArray();
 	}
 
-	async members(uid: string, teamId: string, canSeeAll: boolean, { offset, count }: IPaginationOptions = { offset: 0, count: 50 }, { query }: IQueryOptions<ITeam>): Promise<IRecordsWithTotal<ITeamMemberInfo>> {
+	async members(uid: string, teamId: string, canSeeAll: boolean, { offset, count }: IPaginationOptions = { offset: 0, count: 50 }, { query }: IQueryOptions<IUser>): Promise<IRecordsWithTotal<ITeamMemberInfo>> {
```

Comment:
I think you can use a `Partial<FindOneOptions<IUser>>` here to avoid the casting on line 491.

Reply:
That works, but I'd need to cast the query `param` received by the endpoint (in the `teams.ts` file) 🤔

Reply:
`ping` @sampaiodiego